Makefile

```diff
@@ -1,13 +1,14 @@
 # Building cluster-sanity container
 IMAGE_BUILD_CMD ?= "docker"
```

Comment:
Please change to 
`IMAGE_BUILD_CMD = $(shell which podman 2>/dev/null || which docker)`